src/android/src/main/java/org/pgsqlite/SQLitePlugin.java

```diff
@@ -867,8 +863,9 @@ private int countRowsAffectedCompat(QueryType queryType, String query, JSONArray
             Matcher tableMatcher = UPDATE_TABLE_NAME.matcher(query);
             if (tableMatcher.find()) {
                 String table = tableMatcher.group(1);
+                SQLiteStatement statement = null;
```

Comment:
now we close statements as it supposed to be done.
Probably workaround for android is not needed anymore
At least I delete that code for my production version and it works just fine so far.